diff --git a/igc2kmz/__init__.py b/igc2kmz/__init__.py
index 9c04622..2c33310 100644
--- a/igc2kmz/__init__.py
+++ b/igc2kmz/__init__.py
@@ -45,7 +45,7 @@ else:
     MULTIPLICATION_SIGN = 'x'
     UP_TACK = 'n/a'
 
-BASE_DIR = os.path.normpath(os.path.join(os.path.dirname(__file__), '..'))
+BASE_DIR = "/usr/share/python-igc2kmz"
 
 
 def make_table(rows, bgcolors='#dddddd #ffffff'.split()):
diff --git a/setup.py b/setup.py
index 733af5b..aa20aa9 100644
--- a/setup.py
+++ b/setup.py
@@ -40,6 +40,6 @@ also comes with frontend scripts
                                    "igc2task.py", 
                                    "leonardo2kmz.py", 
                                    "olc2gpx.py"]],
-    package_data={'igc2kmz': ['images/*']},
+    data_files=[('share/python-igc2kmz/images/', ['images/paraglider.png', 'images/pixel.png'])]
     )
 
